src/main/java/Duke.java

```diff
@@ -1,10 +1,65 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static void printList(Task[] list, int counter){
+        for(int i = 0; i < counter; i++){
+            System.out.print(i + 1 + ".");
+            printTask(list[i]);
+        }
+    }
+    public static void printTask(Task t){
```

Comment:
`printTask` is a functionality tied to `Task`, so I think making it a method of `Task` would be more logical.

Reply:
Good idea!